bibtex-actions.el

```diff
@@ -693,8 +694,10 @@ With prefix, rebuild the cache before offering candidates."
 With prefix, rebuild the cache before offering candidates."
   (interactive (list (bibtex-actions-select-refs
                       :rebuild-cache current-prefix-arg)))
- (bibtex-completion-show-entry
-  (bibtex-actions--extract-keys keys-entries)))
+ (let ((bibtex-files (seq-concatenate 'list
+                                      bibtex-actions-bibliography
+                                      (bibtex-actions--local-files-to-cache))))
+  (mapc (lambda (key) (bibtex-find-entry key t nil t)) (bibtex-actions--extract-keys keys-entries))))
```

Comment:
As an upside this works for local bib files while bibtex-completion based one it repalced didn't.

As a possible down side (I am not sure) that might have unintentionally worked for csl-json files since it is regex based, while I don't think this will.

Reply:
I doubt it will either.

BTW: you may have noticed I renamed the project. So this will be one large merge conflict now :-)